### PR TITLE
DOCSUP-39590-doc-fixes

### PR DESCRIPTION
**Pull request by lehasm**
## Description

- Examples with russian changed
- Dash added to every "read-only"
- "Requester's" mentioning removed (Requester's guide).
- True and False wrapped with quotas.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation and examples improvement (changes affected documentation and/or examples)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/client/owner.py [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+
+class Owner(BaseTolokaObject):
+    """Parameters of the customer who created an object.
+
+    Attributes:
+        id: Customer ID.
+        myself: An object accessory marker.
+            Possible values:
+                * `True` — An object is created by the customer whose OAuth-token is used in the request.
+                * `False` — An object does not belong to the customer whose OAuth-token is used in the request.
```
Comment (alexandervnuchkov) [inline src/client/owner.py:13]: No need for a hyphen here, should be `OAuth token`

File: src/client/training.py [hunk 2]
```diff
@@ -30,34 +30,34 @@ class Training(BaseTolokaObject):
...
             You can use HTML markup inside `public_instructions`.
         metadata: A dictionary with metadata.
         owner: The training owner.
-        id: The ID of the training. Read only.
-        status: The training status. Read only.
-        last_close_reason: A reason why the training was closed last time. Read only.
-        created: The UTC date and time when the training was created. Read only.
-        last_started: The UTC date and time when the training was started last time. Read only.
-        last_stopped: The UTC date and time when the training was stopped last time. Read only.
+        id: The ID of the training. Read-only field.read
```
Comment (alexandervnuchkov) [inline src/client/training.py:55]: Should `read` be present here after `field.`?

**Review by alexandervnuchkov — CHANGES_REQUESTED** [review]

File: src/client/owner.py (hunk 1 above)
Comment (lehasm) [reply src/client/owner.py:13]: Fixed

File: src/client/training.py (hunk 2 above)
Comment (lehasm) [reply src/client/training.py]: Certainly, not.

**Review by alexdrydew — APPROVED** [review]

**Pull request merged**